test/replcompletions.jl

```diff
@@ -457,7 +457,6 @@ try
     end
 finally
     @test pop!(LOAD_PATH) == path
-    rm(path, recursive=true)
```

Comment:
This test is literally for the case "even though the path do no exist" (sic)

It doesn't, but it was tripping up my attempts at testing locally so it ended up on all my branches today.